crates/glaredb/src/local.rs

```diff
@@ -136,9 +148,9 @@ impl LocalSession {
         println!("GlareDB (v{})", env!("CARGO_PKG_VERSION"));
         println!("Type \\help for help.");
 
-        let info = match (&self.opts.metastore_addr, &self.opts.local_file_path) {
-            (Some(addr), None) => format!("Persisting catalog on remote metastore: {addr}"),
-            (None, Some(path)) => format!("Persisting catalog at path: {}", path.display()),
+        let info = match (&self.opts.metastore_addr, &self.opts.data_dir) {
+            (Some(addr), None) => format!("Persisting catalog on remote metastore: {addr}"), // TODO: Should we continue to allow this?
```

Comment:
Let's keep allowing this. It might be handy to run some quick commands backed by a self-hosted metastore.